### PR TITLE
pipewire: 0.3.59 -> 0.3.60

### PR DESCRIPTION
**Pull request by K900**
###### Description of changes

Lots of fixes all over.

https://gitlab.freedesktop.org/pipewire/pipewire/-/releases/0.3.60

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/pipewire/default.nix [hunk 1]
```diff
@@ -107,10 +106,10 @@ let
       # Place SPA data files in lib output to avoid dependency cycles
       ./0095-spa-data-dir.patch
 
-      # remove when updating to 0.3.60
-      (fetchpatch { # filter-chain: iterate the port correctly
-        url = "https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/94a64268613adac8ef6f3e6c1f04468220540d00.patch";
-        sha256 = "sha256-IDTB7NgadgR3vKv97Nvd9pBfnOnMi21YsvLdD1Ew7HE=";
+      # FIXME: remove in 0.3.61
+      (fetchpatch {
```
Comment (jansol) [inline pkgs/development/libraries/pipewire/default.nix:110]: Would be nice to add the message of the commit being backported or at least a short explanation why the patch is needed. (the patch itself is good, I was going to suggest pulling it in before I saw you'd already done that)

Comment (K900) [reply pkgs/development/libraries/pipewire/default.nix:110]: It was recommended on Matrix and also on the DISTROS entry in the changelog.

Comment (jansol) [reply pkgs/development/libraries/pipewire/default.nix:110]: Yeah I just mean it's nice to have the reason or the commit msg repeated here so anyone can see at a glance what it is about without consulting release notes or fetching & reading the patch itself.

Comment (K900) [reply pkgs/development/libraries/pipewire/default.nix:111]: Just pushed it.

**Review by jansol — APPROVED** [review]
Did some basic testing, all good so far.

**Pull request merged**